src/components/elementView/elementTab.tsx

```diff
@@ -161,14 +163,26 @@ class ElementTab extends React.Component<ElementViewProps, ElementViewState> {
       <div>
         <AnchorButton
           className="add"
-          text="Export Bulk"
+          text="Export Table Data"
           icon="import"
           onClick={() => {
             /* handle data based on state */
             this.setState({ fileNameIsOpen: true });
-            console.log(this.state.filters);
+            console.log(this.state.filters)
           }}
         />
+        {this.props.isAdmin ? (
+          <div>
+            <AnchorButton
+              onClick={() => this.props.history.push("/bulk-upload")}
+              className="add"
+              icon="export"
+              text="Upload from CSV file"
```

Comment:
small nit on the wording of the button. Since it is now next to `Add Instance` button, maybe this could say `Add from File`? and make these buttons "minimal" style so they match the `add instance` button.